Code/GraphMol/ChemTransforms/ChemTransforms.cpp

```diff
@@ -628,6 +628,10 @@ ROMol *replaceCore(const ROMol &mol, const ROMol &core,
             if (switchIt) {
               sidechainAtom->invertChirality();
             }
+            // the neighbor count needs to be decremented, in case an additional
+            // bond to the core is present at this chiral center (unlikely!),
+            // because this bond will now be at the end of the neighbor list.
+            --whichNbr;
```

Comment:
Is there a test for this fix?

Reply:
> Is there a test for this fix?

Not in this PR.  I found this same issue in `RCore::extractCoreFromMolMatch` for which there are new tests.  I'll look into using the same structures to test `replaceCore`.  Maybe it's best to comment out the line here and do it in a new PR.